### PR TITLE
async completion. check if need to support the original completion co…

### PR DESCRIPTION
**Pull request by ivanbasov**
fixes https://github.com/dotnet/roslyn/issues/31135

### CODE REVIEW

File: src/EditorFeatures/Core/Implementation/IntelliSense/AsyncCompletion/ItemManager.cs [hunk 1]
```diff
@@ -284,11 +283,24 @@ private static bool IsAfterDot(ITextSnapshot snapshot, ITrackingSpan applicableT
...
 
         private FilteredCompletionModel HandleDeletionTrigger(
+            CompletionTriggerReason filterTriggerKind,
             List<ExtendedFilterResult> filterResults,
             string filterText,
             ImmutableArray<CompletionFilterWithState> filters,
             ImmutableArray<CompletionItemWithHighlight> highlightedList)
         {
+            if (filterTriggerKind == CompletionTriggerReason.Insertion &&
+                    !filterResults.Any(r => r.FilterResult.MatchedFilterText))
```
Comment (jasonmalinowski) [inline src/EditorFeatures/Core/Implementation/IntelliSense/AsyncCompletion/ItemManager.cs]: Align the indenting here.

**Comment by ivanbasov** [comment]
> i feel like this should have some sort of test showing how the behavior changes.

I also expected so. However, it seems there were two changes: if we'd apply one of them, we'd break tests. However, two changes work identical.

We could invent a new one to highlight the difference.

**Comment by CyrusNajmabadi** [comment]
> We could invent a new one to highlight the difference.

Yes.  That's what i meant :)

**Comment by ivanbasov** [comment]
Thank you! Added

---
In reply to: [459480114](https://github.com/dotnet/roslyn/pull/32993#issuecomment-459480114) [](ancestors = 459480114)

**Pull request merged**